extensions/wikia/JsonFormat/JsonFormatService.php

```diff
@@ -1,20 +1,20 @@
 <?php
 
-class JsonFormatService extends \WikiaService {
+use Wikia\JsonFormat\HtmlParser;
+
+class JsonFormatService extends WikiaObject {
 
 	const SIMPLE_JSON_SCHEMA_VERSION = 3;
 	const SIMPLE_JSON_CACHE_EXPIRATION = 14400; //4 hour
 
 	private $htmlParser;
 	private $requestContext;
 
-	function __construct( $htmlParser = null ) {
-		if( $htmlParser == null ) {
-			$htmlParser = new Wikia\JsonFormat\HtmlParser();
-		}
-		$this->requestContext = new RequestContext();
-		$this->htmlParser = $htmlParser;
+	public function __construct( HtmlParser $htmlParser = null ) {
```

Comment:
The only use of this class does not pass this argument:

```php
includes/wikia/api/ArticlesApiController.class.php
981:		$jsonFormatService = new JsonFormatService();
```

Maybe let's just remove it?

And there's:

```
extensions/wikia/Search/classes/IndexService/DefaultContent.php
9:use JsonFormatService;
```

but no use of this class within the file.